linera-base/src/crypto.rs

```diff
@@ -430,6 +430,12 @@ impl CryptoHash {
     pub fn as_bytes(&self) -> &HasherOutput {
         &self.0
     }
+
+    /// Returns the hash of `TestSignable(s)`, for testing purposes.
+    #[cfg(any(test, feature = "test"))]
+    pub fn debug(s: impl Into<String>) -> Self {
```

Comment:
I'd suggest renaming this to `for_testing` or something, to avoid conflict (or rather, confusion) with the `Debug` trait.

Reply:
Makes sense; then we should probably also rename `PublicKey::debug`.